.travis.yml

```diff
@@ -39,13 +39,10 @@ jobs:
     - make types
   - &run-tests
     stage: test
-    name: "Test 3.5"
```

Comment:
while we are at it, can you try adding `3.8-dev` to travis? (3.8 should get released in October so would be good to start testing against it)

(if the build breaks we can create a separate PR)

Reply:
it fails so will have to deal with that separately